src/components/CommentCard.jsx

```diff
@@ -1,10 +1,33 @@
 import { dateTimeSeperator } from "../../utils/function";
+import { useContext, useState } from "react";
+import { UserContext } from "./User";
+import { deleteCommentById } from "../../utils/apicalls";
 
-const CommentCard = ({ comment_id, body, author, votes, created_at }) => {
+const CommentCard = ({
+  comment_id,
+  body,
+  author,
+  votes,
+  created_at,
+  comments,
+  setComments,
+}) => {
   const convertedTime = dateTimeSeperator(created_at);
+  const [isDeleted, setisDeleted] = useState(false);
+
+  const { user } = useContext(UserContext);
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    deleteCommentById(comment_id).then(() => {
+      setisDeleted(true);
```

Comment:
This works nicely.  Could we possibly get some more feedback for the user?  Perhaps an alert? Message deleted?